src/Hint/NewType.hs

```diff
@@ -26,39 +31,110 @@ newtype Foo = Foo Int deriving stock Show
 -}
 module Hint.NewType (newtypeHint) where
 
-import Hint.Type
+import Hint.Type (Idea, DeclHint', Note(DecreasesLaziness), ideaNote, ignoreNoSuggestion', suggestN')
 
-newtypeHint :: DeclHint
+import Data.List (isSuffixOf)
+import "ghc-lib-parser" HsDecls
```

Comment:
I de-qualified these. If you were to import the entire `Hint.Type` there would be name clashes, but this saves all the `Hs.` stuff. Is this fine?

Reply:
Within a single hint module, as long as the tests are good, I'm pretty flexible. It's very isolated.